### ISSUE TITLE
[Web][Accessibility][A11yBlocking][11yMAS][Keyboard Navigation][MAS2.1.1][HighImpact][Interactive controls in Welcome Screen are not accessible by keyboard.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#697]
**User Experience:**
User who rely on keyboard will not be able to access those controls if they are not accessible by keyboard which may lead for external help.
**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
OS Build: Mac OS Sierra 0.12.6
Browser: Safari 11.0.1,IE11
**Attachments:**
[MAS2.1.1_Welcome Screen.zip](https://github.com/OfficeDev/script-lab/files/1449363/MAS2.1.1_Welcome.Screen.zip)

**Repro Steps:**
1. Open the URL: https://script-lab.azureedge.net/
2. Try to Navigate through the page using the Tab key.
3. Verify whether user is able to access all the Interactive controls in the Welcome Screen .

**Current Behavior:**
Interactive Controls in Welcome Screen are not accessible by Keyboard.

**Expected Behavior:**
All Interactive Controls in Welcome Screen should be accessible by Keyboard. So that keyboard user can be able to access all controls without any problem.

**MAS Reference:**
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.1.1
